backstop/assets/demo.css

```diff
@@ -15,6 +14,7 @@ p {
 }
 
 body {
+  margin: 0;
```

Comment:
How this doesn't break test images?

Reply:
body was in the block above this, I just moved it down.